CYBRHardeningCheck/PVWA/PVWAHardeningSteps.psm1

```diff
@@ -177,16 +177,21 @@ Function PVWA_Cryptography_Settings
 			$currentValue = Get-WebConfigurationProperty -pspath $iisPath -filter $filter -name $value
 			if($null -ne $currentValue)
 			{
-				if($currentValue.ToLower() -ne $value)
+				if($currentValue -ne 'yes')
```

Comment:
@NathTheDude I think we should set it with".ToLower()" just in case the user added "Yes" or "YES" in the config file

Reply:
I did not think this logical statement was case sensitive though? Is it?

Reply:
I think it is
Going to merge it to a side branch and apply additional changes - thanks!